### ISSUE TITLE
Files not available in CERNBox since the upgrade

### ISSUE THREAD

**Issue by michzimny** [up2university/kuboxed#32]
Since the upgrade last week, the test Up2U instance cernbox.test.up2university.eu (aka cernbox-alpha) gets broken, as follows, every several hours. I have experienced it around 5 times since last Thursday.

A user logs in and sees no files in "All files". Options "Shared with ..." in the menu works fine and list some files. The following message also appears in "All files":
> Ten folder jest niedostępny, proszę sprawdzić logi lub skontaktować się z administratorem

I see it in Polish, and it can be translated as:

> This folder is unavailable, please check the logs or contact the administrator

I see nothing unusual in CERNBox logs. SWAN works fine at the same time.

The following procedure fixes the problem:
```
boxedctl delete deploy cernbox
boxedctl delete deploy cernboxgateway
boxedctl create -f CERNBOX.yaml
```

After this fix, it works fine, but gets broken again after several hours, or maybe a day later.

What could I check to track the cause?

**Comment by diocas** [comment]
Next time you see it, please let me know. But don't restart it so that I'm able to debug it.
We have the latest version running in some test machine for weeks and so far everything is working, so I need to check what's happening.

**Comment by michzimny** [comment]
OK

**Comment by michzimny** [comment]
@diocas, please have a look now:
https://cernbox.test.up2university.eu/

**Comment by diocas** [comment]
Yes, it's failing for me. I'll try to debug it.

**Comment by diocas** [comment]
We've found what was misconfigured and fixed it. But we need to understand how/why it happened.

**Comment by michzimny** [comment]
Are you able to share the fix? I'm dealing with the same in another instance (based on webng_beta_psnc too).

**Comment by diocas** [comment]
Inside the eos-mgm container, just do `eos vid add gateway cernbox.cernbox.boxed.svc.cluster.local unix`.
Did it work?

**Comment by michzimny** [comment]
Yes. When executing the command, it did bring the files back. Thanks.

I will also also be checking if the issue does not appear anymore.

**Comment by michzimny** [comment]
It seems the issue is not appearing anymore.

I understand that the command above is a workaround, but the cause is still to be investigated.

**Comment by diocas** [comment]
Yes, please leave this open until we find how to properly solve it.

**Comment by michzimny** [comment]
The same happened after the upgrade of CERNBox in production. The solution from https://github.com/up2university/kuboxed/issues/32#issuecomment-467109854 helped in solving it.
